packages/logger/src/Logger.ts

```diff
@@ -172,25 +172,28 @@ class Logger extends Utility implements LoggerInterface {
   /**
    * Represents whether the buffering functionality is enabled in the logger
    */
-  protected isBufferEnabled = false;
+  protected isBufferEnabled = true;
 
+  /**
+   * Whether the buffer should be flushed when an error is logged
+   */
+  protected flushOnErrorLog = true;
   /**
    * Log level threshold for the buffer
    * Logs with a level lower than this threshold will be buffered
+   * Default is DEBUG
    */
-  protected bufferLogThreshold: number = LogLevelThreshold.DEBUG;
+  protected bufferAtVerbosity: number = LogLevelThreshold.DEBUG;
   /**
    * Max size of the buffer. Additions to the buffer beyond this size will
    * cause older logs to be evicted from the buffer
    */
-  readonly #maxBufferBytesSize = 1024;
+  #maxBufferBytesSize = 1024;
```

Comment:
Let's change it to `20480` as discussed [here](https://github.com/aws-powertools/powertools-lambda-typescript/issues/3634#issuecomment-2669420699).